mlflow/server/__init__.py

```diff
@@ -53,8 +53,21 @@ def serve_static_file(path):
 # Serve the index.html for the React App for all other routes.
 @app.route(_add_static_prefix('/'))
 def serve():
-    return send_from_directory(STATIC_DIR, 'index.html')
-
+    # if index.html is present. What should be the condition here?
+    if os.path.exists(os.path.join(STATIC_DIR, "index.html")):
+        return send_from_directory(STATIC_DIR, 'index.html')
+    # text should be the name of the text file I created?
+    text = 'Unable to display MLflow UI - landing page (index.html) not found. \n' \
```

Comment:
Small style nit, could we use a triple-quoted block to avoid needing to specify newlines etc in the string? We can use the `textwrap` Python standard library module to help avoid unwanted indentation:

e.g.

```
# Add import textwrap to the top of the file
import textwrap

...
text = textwrap.dedent("""
    Unable to display MLflow UI - landing page (index.html) not found. 

    You are very likely running the MLflow server using a source installation of
     the Python MLflow package...
     """)
```